vignettes/opt-dead-expr.Rmd

```diff
@@ -30,6 +30,10 @@ speed_up <- function(obj) {
 
 # Dead Expression Elimination
 
+## Idea
+
+The basic idea behind the dead expression elimination optimizer is to remove all those expressions whose result is neither used in the program nor the removal of which affects the result of the code. Therefore this *dead expression elimination* frees up memory, saves computation time and provides a speed boost.
```

Comment:
`program nor the removal of which affects` ~> `program, nor the removal of it affects`
`computation time and provides` ~> `computation time, and provides`